docs/source/refcard.rst

```diff
@@ -63,11 +63,11 @@ Reference Card
 | Wait for task to finish | ``await task_0.complete``                                      |
 +------------------------++----------------------------------------------------------------+
 |                                                                                          |
-+-------------------------------+----------------------------------------------------------+
-| Resume on any Task or Trigger | ``await cocotb.triggers.First(task_0, trigger_1)``       |
-+-------------------------------+----------------------------------------------------------+
-| Resume on all Task or Trigger | ``await cocotb.triggers.Combine(task_0, trigger_1)``     |
-+-------------------------------+----------------------------------------------------------+
++----------------------------------+-------------------------------------------------------+
+| Resume after any Task or Trigger | ``await cocotb.triggers.select(task_0, trigger_1)``   |
++----------------------------------+-------------------------------------------------------+
+| Resume after all Task or Trigger | ``await cocotb.triggers.gather(task_0, trigger_1)``   |
++----------------------------------+-------------------------------------------------------+
```

Comment:
Add the new `wait` here too?

Reply:
I don't know if it's something that people will typically use. It's usable on it's own, but more useful as the base of `gather` and `select`.

Reply:
Okay, leaving it out is good then.